css/selectmenu.css

```diff
@@ -114,6 +114,7 @@
 	overflow: hidden;
 	position: relative;
 	text-decoration: none;
+	box-sizing: border-box; /* To keep width calculation in percent since WP 5.6 */
```

Comment:
I guess this works for WordPress < 5.6 as well, does it not?

Reply:
It's what I wrote in my latest comment

> * 95% for each WP version / each jQuery UI version / each browser for the languages settings screen
> * 100% for each WP version / each jQuery UI version / each browser for the Wizard steps

Maybe due to changes in how the width is calculated in jQuery UI 1.12.1 updated with WP 5.6. 
As `box-sizing: border-box` forces to include all other sizes - espacially border and padding - in the size of the HTML element, its size is always the same whatever the case.